instrumentation/rpc/RATIONALE.md

```diff
@@ -87,3 +119,30 @@ parameterized sampling. This happens before a span name is chosen. Moreover, a
 future change will allow parsing into a span name from the same type. In other
 words, `RpcRequest` is an intermediate model that must evaluate properties
 before a span exists.
+
+## Why "rpc.error_code"?
+
+## Why not "rpc.error_message"?
```

Comment:
worth to mention that error_code is more likely to be searchable vs error_message?

Reply:
seems I died in the middle of writing `Why "rpc.error_code"?` good catch :)

Reply:
done!